javascript/javascript3/week1/weatherApp/weather.js

```diff
@@ -0,0 +1,144 @@
+      
+        //Wind speed in meter/second
+        getHtmlMarkup('windSpeed').innerHTML = `Wind Speed: ${weatherData.wind.speed} m/sec`;
+        
+        
+        getHtmlMarkup('timeinfo').innerHTML = `Below time is the time in city location ${weatherData.name.toUpperCase()} and not displayed according to your current location`
+        
+        //current Time and date in the selected city, it updates only when the page refresh's
+        const todayDateTimeUtc = todayDate.getTime() + (todayDate.getTimezoneOffset() * 60000);
+        const todayLocationDateTime = new Date(todayDateTimeUtc + (weatherData.timezone * 1000));
+
+        getHtmlMarkup('locationCurrentDate').innerHTML = `Today's Date: ${todayLocationDateTime.toLocaleDateString()}`
+        getHtmlMarkup('locationCurrentTime').innerHTML = `Current Time: ${todayLocationDateTime.toLocaleTimeString("en-US")}`
+       
+        //Sunrise and Sunset time in the city location 
+        const dateTime1 = new Date(weatherData.sys.sunrise * 1000);
+        const dateTime2 = new Date(weatherData.sys.sunset * 1000);
+        const dateTime1Utc = dateTime1.getTime() + (dateTime1.getTimezoneOffset() * 60000);
+        const dateTime2Utc = dateTime2.getTime() + (dateTime2.getTimezoneOffset() * 60000);
+        const sunriseDateTime = new Date(dateTime1Utc + (weatherData.timezone * 1000));
+        const sunsetDateTime = new Date(dateTime2Utc + (weatherData.timezone * 1000));
+
+        getHtmlMarkup('sunrise').innerHTML = `Sunrise : ${sunriseDateTime.toLocaleTimeString("en-US")}`;
+        getHtmlMarkup('sunset').innerHTML = `Sunset : ${sunsetDateTime.toLocaleTimeString("en-US")}`;
+       
+        //embeding the city location on map in the html page
+        const map = getHtmlMarkup('mapLocation')
+        map.src = `https://maps.google.com/maps?q=${weatherData.coord.lat},${weatherData.coord.lon}&output=embed`
+
+    })
+}
+
+//for reference
+// fetch(`https://api.openweathermap.org/data/2.5/weather?q=copenhagen&appid=${apiKey}`)
+//     .then(response => response.json())
+//     .then(weatherData => {
+//         console.log(weatherData);
+//         console.log('The chosen city: ' + weatherData.name);
+//         console.log(`Temperature: ${weatherData.main.temp}`);
+//         console.log(`Icon for the weather type: ${weatherData.weather[0].icon}, \n http://openweathermap.org/img/wn/${weatherData.weather[0].icon}@2x.png`);
+//         console.log(`How clowdy it is: ${weatherData.weather[0].description} - ${weatherData.clouds.all}%`);
+//         console.log(`Feels_Like: ${weatherData.main.feels_like} °C`);
+//         console.log(`Wind Speed: ${weatherData.wind.speed}`);
+//         console.log(`Location Time: ${todayLocationDateTime}`);
+//         console.log(`When sunrise and sunset is \n sunrise: ${new Date(weatherData.sys.sunrise * 1000)} \n sunset: ${new Date(weatherData.sys.sunset * 1000).toLocaleTimeString("en-US")}`);
+//         console.log(map);
+//     })
```

Comment:
**please remove the commented code.**
Note: make sure to remove unwanted **console.log** and **commented codes** before pushing the code

Reply:
Good job @ShravyaGurmitkal  
I have few comments